src/main/java/ToDo.java

```diff
@@ -0,0 +1,18 @@
+public class ToDo extends Task {
+    private final String TO_DO_SYMBOL = "[T]";
```

Comment:
Again, very small comment: should probably be TODO_SYMBOL instead of TO_DO_SYMBOL (since "todo" is generally treated as one work)

Reply:
Agreed. I would change the naming of the constant.